src/fides/common/api/v1/urn_registry.py

```diff
@@ -20,15 +20,6 @@
 
 CONSENT_REQUEST_VERIFY = "/consent-request/{consent_request_id}/verify"
 
-# Privacy Preference URLs
-CONSENT_REQUEST_NOTICES_SERVED = "/consent-request/{consent_request_id}/notices-served"
-NOTICES_SERVED = "/notices-served"
-
-# Reporting endpoints - have records for *all* users
-HISTORICAL_PRIVACY_PREFERENCES_REPORT = "/historical-privacy-preferences"
-CURRENT_PRIVACY_PREFERENCES_REPORT = "/current-privacy-preferences"
-
```

Comment:
These endpoints are already in Plus, but we were importing their URN's from this file